.changeset/healthy-kids-mate.md

```diff
@@ -0,0 +1,9 @@
+---
+'@backstage/cli': patch
+'@backstage/cli-common': patch
+'@backstage/create-app': patch
+---
+
+Add .backstage-version file
```

Comment:
This could use an update now :p

Prolly best to split this up into multiple changesets btw, and have the create-app changeset explain how to get the file created